internal/impl/nats/input.go

```diff
@@ -160,11 +160,13 @@ func (n *natsReader) Connect(ctx context.Context) error {
 	}
 
 	opts = append(opts, authConfToOptions(n.authConf, n.fs)...)
+	defer deleteTempNkeyFile()
```

Comment:
hmm, is this file safe to delete at this moment? Will the lib reads it later?

Reply:
I tested it, only read at connection